src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1783,6 +1866,17 @@ static void glb_update_locked(grpc_exec_ctx *exec_ctx, grpc_lb_policy *policy,
         &glb_policy->lb_channel_connectivity,
         &glb_policy->lb_channel_on_connectivity_changed, NULL);
   }
+
+  // Propagate update to fallback_backend_addresses if we haven't heard any valid 
+  // serverlist from the balancer.
+  if (glb_policy->serverlist == NULL) {
```

Comment:
I don't think this is a problem.  Once we set the serverlist, I don't think we ever unset it.  We may replace it with a new serverlist, but I don't think we ever set it to NULL.